tier0/{{cookiecutter.project_slug}}/COMMUNITY.md

```diff
@@ -1,5 +1,6 @@
 # COMMUNITY.md
 
```

Comment:
Could we move line 8 to line 2 so it is right below the COMMUNITY.md header? Then it is just the table with comments under the Project Members section